packages/p2p/src/components/buy-sell/buy-sell-table.jsx

```diff
@@ -11,7 +11,6 @@ export const BuySellTable = ({ children }) => {
                     <Table.Head>{localize('Advertisers')}</Table.Head>
                     <Table.Head>{localize('Limits')}</Table.Head>
                     <Table.Head>{localize('Rate (1 USD)')}</Table.Head>
-                    <Table.Head>{localize('Payment method')}</Table.Head>
```

Comment:
Why're we removing `Payment method` from the table?

Reply:
We've decided to remove this field from P2P as it'll be a free-form type of input (i.e. can put in e-wallet, paypal, or cake). I believe the design is (going to be) updated to reflect this change.